subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,15 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files) found in inputs/outputs of tasks are handled gracefully from now on: similarly to missing files.
+
```

Comment:
Can we add an example use case here why this is useful? Or what is the change in behaviour?

Reply:
Added, if you think more is needed, please insert a suggestion.